components/Conversations/Detail/ConversationDetail/NoAudioDetected.tsx

```diff
@@ -1,54 +1,50 @@
-import React from 'react';
-import GreyShieldTick from '../Icon/GreyShieldTick';
-import BigSoundIcon from '../Icon/BigSoundIcon';
-import TrashIcon from '../Icon/TrashIcon';
+import React from 'react'
+import GreyShieldTick from '../Icon/GreyShieldTick'
+import BigSoundIcon from '../Icon/BigSoundIcon'
+import { Trash } from 'iconsax-react'
+// import TrashIcon from '../Icon/TrashIcon';
 
 const NoAudioDetected: React.FC = () => (
-  <div className="w-[840px] h-[1829px] relative font-inter">
+  <div className="font-inter relative h-[1829px] w-[840px]">
     <div className="absolute left-[62px] top-[48px] inline-flex items-center gap-[13px]">
-      <div className="flex items-center justify-center w-8 h-8">
-        <div className="relative w-8 h-8">
+      <div className="flex h-8 w-8 items-center justify-center">
+        <div className="relative h-8 w-8">
           <BigSoundIcon />
         </div>
       </div>
-      <div className="text-2xl font-semibold leading-[31px] text-white">
-        No Audio Detected...
-      </div>
+      <div className="text-2xl font-semibold leading-[31px] text-white">No Audio Detected...</div>
     </div>
     <div className="absolute left-[64px] top-[104px] w-[624px] text-[15px] font-normal leading-normal text-[#484848]">
       Transcript will update every ~30s
     </div>
     <div className="absolute left-[549px] top-[48px] inline-flex items-center gap-4">
-      <div className="flex items-center justify-center w-6 h-6 opacity-40">
-        <div className="relative w-6 h-6">
-          <TrashIcon />
+      <div className="flex h-6 w-6 items-center justify-center opacity-40">
+        <div className="relative h-6 w-6">
+          <Trash variant="Bold" className="group-hover:text-destructive text-primary" size={24} />
```

Comment:
ah we have destructive here all good

Reply:
oh lol didn't notice that one - still not sure what it does though